### DISCUSSION TITLE
categories/%EA%B0%9C%EB%B0%9C/%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC/WebSocket-%EC%B1%95%ED%84%B0-2-%EC%9B%B9%EC%86%8C%EC%BC%93-%ED%94%84%EB%A1%9C%ED%86%A0%EC%BD%9C/

### DISCUSSION THREAD

**Discussion by giscus** [redniche/redniche.github.io#7] [Announcements]
# categories/%EA%B0%9C%EB%B0%9C/%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC/WebSocket-%EC%B1%95%ED%84%B0-2-%EC%9B%B9%EC%86%8C%EC%BC%93-%ED%94%84%EB%A1%9C%ED%86%A0%EC%BD%9C/

웹소켓 프로토콜 (WebSocket Protocol)

https://yechankun.github.io/categories/%EA%B0%9C%EB%B0%9C/%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC/WebSocket-%EC%B1%95%ED%84%B0-2-%EC%9B%B9%EC%86%8C%EC%BC%93-%ED%94%84%EB%A1%9C%ED%86%A0%EC%BD%9C/

**Comment by yechankun** [comment]
https://hpbn.co/websocket/#websocket-protocol
의 내용을 번역한 것입니다.
